src/sentry/integrations/msteams/webhook.py

```diff
@@ -200,6 +202,55 @@ def handle_member_removed(self, request):
         integration.delete()
         return self.respond(status=204)
 
+    def make_action_data(self, identity, data):
+        action_data = {}
+        if data["actionType"] == "unresolve" or data["actionType"] == "unignore":
+            action_data = {"status": "unresolved"}
+            analytics.record("integrations.msteams.unresolve", actor_id=identity.user_id)
+        elif data["actionType"] == "resolve":
+            status = data["resolveInput"]
+            status_data = status.split(":", 1)
```

Comment:
Hmmm, what are we splitting on here? I'm not seeing anything [in the tests](https://github.com/getsentry/sentry/pull/20111/files#diff-ace9eaab1b4977698aaa4b33a01ddc6eR198) to suggest that we need to split.

Reply:
For resolve, it can be something like "resolve:inCurrentRelease" or "resolve:inNextRelease". I didn't know how to mock releases for testing, so I didn't account for either of those cases.